unrpyc.py

```diff
@@ -346,10 +346,24 @@ def main():
 
     args = ap.parse_args()
 
+    # Catch impossible arg combinations with clear info, so they do not produce unclear
+    # errors or fail silent
+    if (args.no_pyexpr or args.comparable) and not args.dump:
+        raise ap.error(
+            "Arguments 'comparable' and 'no_pyexpr' are not usable without 'dump'.")
+        return
+
+    if ((args.try_harder or args.dump)
+            and (args.write_translation_file or args.translation_file or args.language)):
+        raise ap.error(
+            "Arguments 'try_harder' and/or 'dump' are not usable with the translation "
+            "feature.")
+        return
```

Comment:
This return statement is superfluous (as the one above)

Reply:
😊 
Thanks!